BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -966,3 +966,4 @@
 "PleaseRestartRegistrationOrTryLoggingInYouMayAlreadyHaveAnAccount" = "Please restart registration or try logging in. You may already have an account.";
 "RemovePasskey" = "Remove passkey";
 "PasskeyRemoved" = "Passkey removed";
+"AutofillBiometricUnlockForAccountDisabledPendingVerificationOfMaterPassword" = "Autofill biometric unlock for this account is disabled pending verification of master password";
```

Comment:
⚠️ This string should already exist since this was used in MAUI.

https://github.com/bitwarden/ios/blob/f23f7cba155a2cf441e3cabc26dff28957864ae7/BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings#L505

Reply:
oh snap! good catch, thanks!